frontend/src/helpers/row.helper.ts

```diff
@@ -0,0 +1,15 @@
+
+export interface DataRowModel {
+  rowKey: string;
+  rowData: (includeNulls: boolean, includeKey: boolean) => any[];
+}
+export class DataRowHelper {
+  public static dataRowsAsValueDict<T>(
```

Comment:
Z sumie to jest trochę dziwna funkcja która robi to, że konwertuje kolekcję `DataRow[]` do słownika  o postaci `dataRowKey: dataRowData`. 

Używam ją, ponieważ model harmonogramu jest słownikiem i z tego powodu logika też jest tworzona na podstawie słownika. Ale czasami potrzebuję utworzyć model na podstawie `DataRow[]` i wtedy potzeba konwertować to do takiego słownika, jakimi są sekcje w harmonogramie